win32_event_log/datadog_checks/win32_event_log/data/conf.yaml.example

```diff
@@ -228,35 +228,42 @@ instances:
     ##
     ## Setting this option to `false` is only supported on Agent versions 7 and above.
     #
-    # legacy_mode: true
+    legacy_mode: false
```

Comment:
This will make new users use the new implementation while old users get the default true value